### ISSUE TITLE
Create a page for screenreaders to understand the graph

### ISSUE THREAD

**Issue by avantitandon** [Breaking-Bias/CashApp_FTL#131]
**ACCESSIBILITY**
I am a screenreader, I want to understand the results of the graph.

- [x] Page should be accessible only to screenreader
- [x] Have text to explain big numbers, include percentage increased.
